indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -266,6 +268,16 @@ Set<String> taskIds()
       return tasks.keySet();
     }
 
+    void setShutdownEarly()
```

Comment:
Should these methods be renamed to `setHandoffEarly()` to correspond to the API and the method in `SupervisorManager` / `Supervisor` classes?